src/Post.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+var x=1
+
+function Post(){
+let [count,setcounter] = useState(0)
+if(count>=100){
+    alert(`You Win!`)
+}
+function setval(){
+    if(count<10){
+        alert("You can't afford that!")
+    }
+     else {
+count=count -10;
```

Comment:
Remember, we've got specific methods to update state, and using those methods is what lets React update and render the page for us.